types/protocols/protocols-tests.ts

```diff
@@ -0,0 +1,6 @@
+import protocols = require('protocols');
+
+protocols('git+ssh://git@some-host.com/and-the-path/name'); // $ExpectType string
```

Comment:
This is incorrect. Expected type needs to be string[]

Reply:
Sorry :smiley: Should be good now.